.github/workflows/build-latest.yml

```diff
@@ -46,4 +46,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04, ubuntu-22.04, macos-12, macos-14, windows-2022]
+        os:
+          [
+            ubuntu-22.04,
+            ubuntu-24.04,
+            macos-13,
+            macos-14,
+            windows-2022,
```

Comment:
Should that be `macos-15`?

Reply:
No, not yet. Soon, yes.

Reply:
Ok, but the windows-2022 appears twice (there and below).